gshoppingflux/gshoppingflux.php

```diff
@@ -1634,7 +1634,7 @@ public function renderInfo()
 		));
 	}
 
-	public function customGetNestedCategories($shop_id, $root_category = null, $id_lang = false, $active = true, $groups = null, $use_shop_restriction = true, $sql_filter = '', $sql_sort = '', $sql_limit = '')
```

Comment:
sql_sort and sql_limit are not used either, except for debugging purposes.. do you think removing the filter option is really relevant? does it optimize the whole process significantly ? thx anyway :)